specification/09-coveio_abi.adoc

```diff
@@ -52,8 +52,35 @@ identified by `stream_id`.
 struct sbiret sbi_covh_disconnect_device()(unsigned long device_id);
 -----
 
-==== Device Interface Assignment
+==== TVM Memory Management
 
+[#sbi_covh_add_tvm_device_region()]
```

Comment:
Do we need corresponding `remove` style API?

Reply:
Would it make sense to e.g. remove one TDI MMIO region from the TVM, while keeping the TDI bound to the TVM? Wouldn't the TDI be mostly useless for the TVM at that point? If that's so, I think we should not allow the TVM to do that and only have it reclaim the TDI by unbinding it (`sbi_covh_unbind_interface`) if it wants to unmap those regions from the TVM.
What do you think?

Reply:
It is more for the secure clean up.

I expect that the TVM may request the VMM to unbind the device. Then the VMM needs to **remove the device related MMIO and DMA**, from the TVM domain, by calling TSM interface.

Later, the TVM can verify if VMM has done that correctly, by calling TSM interface to check.

Reply:
Makes a lot of sense to me, thanks. I added a `sbi_covh_reclaim_tvm_interface_region` API.